test/integration/e2e/index.js

```diff
@@ -49,6 +49,7 @@ describe('basic E2E tests', () => {
     let aliceRegTxId;
 
     let aliceUser;
+    let bobContactRequest;
```

Comment:
Why do we duplicate here our devnet e2e tests?

Reply:
we need tests for verification dapi and others components on a regular basis, ideally is on every commit. we don't need to verify functional tests only after deployment in devnet.